packages/terra-overlay/src/terra-dev-site/test/overlay/Overlay/OnRequestCloseOverlay.test.jsx

```diff
@@ -32,8 +32,10 @@ class OverlayExample extends React.Component {
           <br />
           <p>Close by clicking inside the overlay or pressing the ESC key.</p>
         </Overlay>
-        <button type="button" id="trigger_container" onClick={this.handleTriggerOverlay}>Trigger Container Overlay</button>
-        <button type="button" id="trigger_fullscreen" onClick={this.handleTriggerFullScreenOverlay}>Trigger Fullscreen Overlay</button>
+        <div>
```

Comment:
I am confused by this change. I thought this PR added an overlay prop such that overlays should be constructed like:
```diff
+ const overlayComponent = (
+    <Overlay isOpen={this.state.show} isRelativeToContainer={this.state.relative}>
+          // content displayed in overlay
+    </Overlay>
+  );

- <OverlayContainer>
+ <OverlayContainer overlay={overlayComponent}>
-    <Overlay isOpen={this.state.show} isRelativeToContainer={this.state.relative}>
-          // content displayed in overlay
-    </Overlay>
      <button>Trigger Container Overlay</button>
      <button>Trigger Fullscreen Overlay</button>
 </OverlayContainer>
```

Reply:
updated the examples and read me files with use of [overlay prop](https://github.com/cerner/terra-core/pull/2249/commits/4dd8524ffaaec55fcb3f9c13fedde57d4f555908)